source/client/output_formatter_impl.cc

```diff
@@ -292,7 +304,8 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   fortio_output.set_numthreads(number_of_connections);
 
   // Get the result that represents all workers (global)
-  const auto& nh_global_result = getGlobalResult(output);
+  auto nh_global_result_status = getGlobalResult(output);
+  const auto& nh_global_result = nh_global_result_status.value();
```

Comment:
I know that you and Otto had a conversation here about whether or not to include a line here returning the status if it's not `ok()` and deciding not to.  I think I disagree with the conclusion.  Could we please return the status if it's not ok here?

Reasoning:
1. I don't think it's a good idea to code with perfect understanding of a function's error state, because functions aren't static - they change over time.  While it may not be possible for getGlobalResult to actually return an error _yet_, it might be later, and now this code wouldn't operate as expected.

2. Removing lines just because covering them isn't possible is a distorted incentive.  Coverage should never discourage us from doing the right thing.

Reply:
This comment drew me into looking into this once more; what I think might be appropriate is for `getGlobalResult() is` to return an `absl:::optional<const nighthawk::client::Result>` instead of a status: I can't anticipate failures modes where an exception wouldn't be appropriate, and not having a global result in the output is business as usual in terms of transforming to Fortio output.